### PR TITLE
Day8 and extra

### PR DESCRIPTION
**Pull request by dat-nnguyen**
quite late

### CODE REVIEW

File: Day8.py [hunk 1]
```diff
@@ -1 +1,57 @@
-# DAY 8 - DICTIONARIES 
\ No newline at end of file
+# DAY 8 - DICTIONARIES
```
Comment (tuang3142) [inline Day8.py:1]: gud to go 🍀

File: extra.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#EXTRA EXERCISE
```
Comment (tuang3142) [inline extra.py]: don't make a new file. edit files in the `extra/` folder in stead.

**Review by tuang3142 — COMMENTED** [review]
pls fix

File: Day8.py (hunk 1 above)
Comment (dat-nnguyen) [reply Day8.py:1]: bro I finished that

File: extra.py (hunk 2 above)
Comment (dat-nnguyen) [reply extra.py]: bro I synced change 2 days ago

**Pull request merged**